src/EntityComponentManager.cc

```diff
@@ -225,6 +232,8 @@ class ignition::gazebo::EntityComponentManagerPrivate
 
   /// \brief See above
   std::unordered_map<Entity, Entity> oldToClonedCanonicalLink;
```

Comment:
Can we add `public` to this variable and the `oldModelCanonicalLink` variable above? It looks like I forgot to add those visibility flags in when working on #959.

Reply:
Done in https://github.com/ignitionrobotics/ign-gazebo/pull/1062/commits/7fa0aa2b95d00f2dff94611c75932d455a4eafa3